cognite/client/_proto/data_points.proto

```diff
@@ -4,9 +4,15 @@ package com.cognite.v1.timeseries.proto;
 
 option java_multiple_files = true;
 
+message Status {
+    int64 code = 1;
+    string symbol = 2;
+}
+
```

Comment:
There’s a separate file for users on protobuf<4 that also needs updating. I guess Status isn’t used yet so no real worries